Create swarm

erickalozon opened the pull request:
Started code for manual particle swarm optimization

erickalozon committed:
Create swarm

erickalozon committed:
Update swarm

erickalozon committed:
Update swarm

zihehuang commented on swarm:
```diff
@@ -0,0 +1,80 @@
+        %Check if boundary conditions are violated element by element
+        if naturalperiod1 <=25 || naturalperiod2<=25 ||naturalperiod3<=25|| pitchoffset > 8 || GM(i)<=0 || wt_ballast(i)<0 ||...
+            swarm(i,1) <0 || swarm(i,2) <0 || swarm(i,3) <0 || swarm(i,4) <0
+            cost(i,1) = 1000000000;
```
```suggestion
            cost(i,1) = 10e9;
```

dstep16 committed:
Update swarm

Lines 10-13: Updated D and T range slightly, showed better graphing results when I did this. 
Line 51: Updated cost penalty to 2.5E7, this showed good results on the graph (similar scale as the non-penalized cost but still higher)
Lines 74-76 and Lines 86-88: Added 3D scatter plot to plot each swarm generation and G-best

changjas committed:
Updated boundary conditions to the minimum we set
